sema/subyt/sources.py

```
@@ -207,7 +208,15 @@ def __init__(self, pattern: str, pattern_root_dir: str = "."):
         self._pattern: str = pattern
         self._init_sourcefiles(
-            [f for f in self._collection_path.glob(pattern) if f.is_file()]
+            [
+                f
+                for f in getMatchingGlobPaths(
+                    self._collection_path,
+                    includes=[pattern],
+                    onlyFiles=True,
+                    makeRelative=False,
+                )
+            ]
```

Ruff (0.8.2)

Unnecessary list comprehension (rewrite using `list()`)

Rewrite using `list()`

(C416)